drivers/serial/uart_ns16550.c

```diff
@@ -321,13 +323,19 @@ static int uart_ns16550_configure(struct device *dev,
 	unsigned int old_level;     /* old interrupt lock level */
```

Comment:
This should be removed, along with the calls to `irq_lock()` and `irq_unlock()` which have been superseded by use of ths spin lock.

Reply:
Thanks, I'll update this!